Userland/Libraries/LibGfx/ClassicWindowTheme.cpp

```diff
@@ -183,15 +198,16 @@ void ClassicWindowTheme::paint_notification_frame(Painter& painter, const IntRec
 IntRect ClassicWindowTheme::frame_rect_for_window(WindowType window_type, const IntRect& window_rect, const Gfx::Palette& palette, int menu_row_count) const
 {
     auto window_titlebar_height = titlebar_height(window_type, palette);
+    auto border_thickness = palette.window_border_thickness();
 
     switch (window_type) {
     case WindowType::Normal:
     case WindowType::ToolWindow:
         return {
-            window_rect.x() - 4,
-            window_rect.y() - window_titlebar_height - 5 - menu_row_count * menubar_height,
-            window_rect.width() + 8,
-            window_rect.height() + 9 + window_titlebar_height + menu_row_count * menubar_height
+            window_rect.x() - border_thickness,
+            window_rect.y() - window_titlebar_height - border_thickness - menu_row_count * menubar_height,
+            window_rect.width() + (border_thickness * 2),
+            window_rect.height() + (border_thickness * 2) + window_titlebar_height + menu_row_count * menubar_height
```

Comment:
Why are these numbers now all the same when there previously was an offset of 1 for the y values?